draft-ietf-rats-architecture.md

```diff
@@ -1458,27 +1486,23 @@ The following example illustrates a hypothetical Background-Check Model solution
         |                |                |               |
         ~                ~                ~               ~
         |                |                |               |
-     time(HR_a)<---------+-------------time(HR_v)------>time(HR_r)
+     time(HR_a)<----time(HT)-----------time(HR_v)----->time(HR_r)
         |                |                |               |
      time(EG_a)          |                |               |
-        |----Evidence{time(EG_a)}-------->|               |
-        | {Handle1,time(EG_a)-time(VG_a)}|                |
+        |--E{H,time(VG_a)}--------------->|               |
```

Comment:
Problems:
1) the term E is not defined, only Evidence
2) You cannot convey time(VG_a) without synchronized clocks
3) time(EG_a) is left in line 1491 but then not used

Reply:
1.) As far as I understand it, this seems to be a proposal simplify the layout of the diagram.

The current state evolved into a representation that was neither readable nor compressible:
  
```
     time(EG_a)
        |----Evidence{time(EG_a)}
        | {Handle1,time(EG_a)-time(VG_a)}
```
Whatever the solution in the end, this has to be fixed, I think (the { vs ( syntax is never introduced and not intuitive).

2. true, we have to walk through the sequence again together, I think

3. It still happens. Is it okay to not illustrates what happens, but is not conveyed in a Conceptual Message? I have some doubts that "used" only is restricted to conveyance or am I missing something obvious here?

Reply:
In the snippet in Henk's comment there's two {} blocks on the same message, which should never happen.
{} means "includes (inside)".  That is Evidence{time(EG_a)} means that time(EG_a) is inside the signed Evidence.   I completely agree it would be good to explain the notation before or in the first example where it's used.